libs/designsystem/src/lib/kirby.common.ts

```diff
@@ -102,4 +102,8 @@ export const providerDeclarations: any[] = [
   AlertHelper,
   ToastHelper,
   ToastController,
+  BreakpointHelperService,
+  LoadingOverlayService,
+  ResizeObserverFactory,
+  ResizeObserverService,
```

Comment:
Isn't this a left-over from the old {N}ativescript days, shouldn't the contents of this file simply be inlined into the `kirby.module.ts`-file instead (and this file then be deleted) - instead of adding to it?

Reply:
Fixed